src/components/AddTodoForm.js

```diff
@@ -2,6 +2,7 @@ import { useState } from "react";
 import React from "react";
 import InputWithLabel from "./InputWithLabel";
 import styles from "./AddTodoForm.module.css";
+import PropTypes from "prop-types";
 
 function AddTodoForm({ onAddTodo }) {
```

Comment:
It doesn't look like you're using the new `onAddTodo` in the form. Look at the `onSubmit` prop.

Reply:
If I replace {handleAddTodo} with {onAddTodo} on the onSubmit prop it adds an empty todo to the list. I don't understand.

Reply:
YOu're that's my bad but I figured it out :)